### ISSUE TITLE
Add RaspberryMatic to DockerHub

### ISSUE THREAD

**Issue by ScumbagSteve** [jens-maus/RaspberryMatic#1194]
I am currently using a VM to run RaspberryMatic but would like to switch to a Docker.
If it could be added to RaspberryMatic, it would be a charm to use.

Unfortunately I don't have the knowledge to set it up manually from within UNRAID.

Regards

**Comment by jens-maus** [comment]
I am sorry, but RaspberryMatic is already available as a docker on the GitHubs' own container registry:

https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Docker-OCI

So what is wrong with that?

**Comment by ScumbagSteve** [comment]
Hi Jens,

I think a lot of people in the community are used to just "one click" install a docker.
I know it's kind of lazy, but if it isn't a big effort to maintain I think it's a great service.

I tried installing the docker manually in unraid, however as I don't know for sure how all this works I am careful with messing around in the terminal of my server. 
I got it to run, but after restarting my docker image got orphaned somehow.
